Meta/serenity.sh

```diff
@@ -101,7 +101,8 @@ if [ "$TARGET" != "lagom" ] && [ -n "$1" ]; then
 else
     TOOLCHAIN_TYPE="${SERENITY_TOOLCHAIN:-"GNU"}"
 fi
-if ! [[ "${TOOLCHAIN_TYPE}" =~ ^(GNU|Clang)$ ]]; then
+[[ "${TOOLCHAIN_TYPE}" = "clang" ]] && TOOLCHAIN_TYPE='Clang'
+if ! [[ "${TOOLCHAIN_TYPE}" =~ ^(GNU|Clang|clang)$ ]]; then
```

Comment:
Why is this necessary?

Reply:
This comes from the tasks.json. The build directory is `Build/x86_64clang`. From the input value I am appending the compiler name to the build directory and also passing that same value to the `Meta/serenity.sh run` command-line. 

There are only two valid compilers `Clang` and `GNU`. So if the compiler is _clang_, I am transforming it to Clang so that it is considered valid.

Reply:
If you remove it, then it will fail as invalid compiler. And making it case-insensitive everywhere doesn't make sense and could be error-prone.

Reply:
Hi @BertalanD, Could you please re-review the PR? Also I have removed `clang` from line 105 in b94c1733550ee34b50ee96def3f358f579f93412